grpc/src/main/java/RouterServer.java

```diff
@@ -107,8 +107,8 @@ private void start() throws IOException {
                 .addService(new RouterImpl(graphHopper, ptRouter, matrixAPI, gtfsLinkMappings, gtfsRouteInfo, gtfsFeedIdMapping))
                 .addService(ProtoReflectionService.newInstance())
                 .maxConnectionAge(10, TimeUnit.SECONDS)
-                .maxConnectionAgeGrace(10, TimeUnit.SECONDS)
-                .executor(Executors.newFixedThreadPool(16))
+                .maxConnectionAgeGrace(30, TimeUnit.SECONDS)
+                .executor(Executors.newFixedThreadPool(3))
```

Comment:
Was there some other config (maybe xml?) where we could tune threads like listener vs worker vs etc? If so, how do these interact?

Reply:
For the old http version, you would set Jetty server settings in the main graphhopper config file (yaml), but for the new grpc version, all server-level settings are set here in the code. So basically, what's in these few lines is all that's used to tune the server. There's loads more that can be set: https://grpc.github.io/grpc-java/javadoc/io/grpc/netty/NettyServerBuilder.html